styles/asciidoc.atom-text-editor.less

```diff
@@ -87,6 +87,10 @@ atom-text-editor, :host {
         font-size: @font-size * .8;
         color: mix(green, @syntax-text-color, 50%);
       }
+
+      &.meta.attrlist {
```

Comment:
Try to avoid custom style, use the 'default' code create a better cross-theme support.

I will try you're PR to see the rendering.